docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -45,6 +45,13 @@ finer, more granular time buckets, which are ideal for this type of analysis. If
 your detectors use <<ml-count-functions,count>> or <<ml-rare-functions,rare>>
 functions, set the interval to the same value as the bucket span.
 
+If your {dfeed} uses aggregations, then the **Anomaly Explorer** in {kib} cannot 
+plot and display an anomaly chart for the job. If model plot is disabled, then 
```

Comment:
Nit - I'd move the word `can` to read `If model plot is disabled, then neither can the Single Metric Viewer plot and display...`.